templates/deployment.yaml

```diff
@@ -0,0 +1,65 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: metadata-service
+  labels:
+    k8s-app: metadata-service
```

Comment:
Small nit, but can we make sure we add all of the Helm labels to our objects. Something like:
```
    helm.sh/chart: {{ $.Chart.Version | quote }}
    app.kubernetes.io/name: {{ template "common.names.fullname" . }}
    app.kubernetes.io/instance: {{ $.Release.Name | quote }}
    app.kubernetes.io/managed-by: {{ $.Release.Service | quote }}
```

Reply:
I've fixed this by using the bitnami common.labels.standard include.